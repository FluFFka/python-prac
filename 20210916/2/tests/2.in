[10, -52, 489465, -465132154647896, 500000000000000000000000000000000000000000000000000000000000000000000000000000, 3, 4, 9, -8, -9, -2, -333]
