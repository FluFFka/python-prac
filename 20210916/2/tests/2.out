[-465132154647896, -333, -52, -9, -8, -2, 3, 4, 9, 10, 489465, 500000000000000000000000000000000000000000000000000000000000000000000000000000]
